tests/test_wcs.py

```diff
@@ -2467,21 +2467,19 @@ def test_fittedsipwcs():
     """
     import astropy.io.fits as fits
 
-    if __name__ == '__main__':
-        test_tags = all_tags
-    else:
-        test_tags = ['TAN', 'SIP', 'TAN-PV']
+    test_tags = all_tags
 
     tol = {  #(arcsec, pixels)
-        'HPX': (5.0, 0.01),
-        'TAN': (5.0, 0.01),
-        'TSC': (5.0, 0.01),
-        'STG': (5.0, 0.01),
+        'HPX': (5.0, 0.02),
+        'TAN': (5.0, 0.02),
+        'TSC': (5.0, 0.02),
+        'STG': (5.0, 0.02),
         'ZEA': (10.0, 0.1),
         'ARC': (5.0, 0.02),
         'ZPN': (2500.0, 2.5),
```

Comment:
Maybe worth commenting here (or below where you delve deeper) that this particular image spans a crazy large portion of the sky.  You can open it in ds9 and see.  
The declination goes from about -18 in the upper right corner to -84 in the lower left, which is more than 60 degrees.  RA ranges from 17:50 to 21:44 hours, so more than 45 degrees of RA, which is actually less than 45 degrees across the sky, but still pretty huge.  It's not necessarily surprising that other projections, even with a SIP step will not work so well.
Making this even more difficult, the projection center of the lambert projection for this particular image is off the image.  They project it at the south pole, which is at CRPIX (-183,22).  The image itself is size (192,192), so it's almost a full image size off the left edge of the image, and down near the bottom.  So when you try to use a ZEA projection here, centered at the center of the image, it's not actually a good match to the original, so it doesn't work.  It's not obvious why it would be a little worse than TAN, but since the center of the projection is not close to the original, I don't think we should expect the ZEA to necessarily be better either.
Anyway, all this is just to say that I think having a vv large tolerance for this image is fine and not indicative of a problem anywhere.  It's unfortunate that this test image is so crazy.  It would have been cute to showcase the ZEA-SIP working here.  But unless you want to add an option to give a manual projection center (which I don't think we care about), it looks like we shouldn't expect anything to do a good job on this one.

Reply:
BTW, here is a screen shot of that in ds9:
![image](https://user-images.githubusercontent.com/623887/88357734-4d7c2400-cd3a-11ea-86df-9d8698b1df61.png)

Reply:
Looks like it's -39 to -78 to me (staying within the 192 pix bounding box) (?), but yeah, still huge.

I dug a little further and found section 5.1.7 of [Calabretta and Greisen II](https://www.aanda.org/articles/aa/pdf/2002/45/aah3860.pdf).  Looks like ZPN is a generalization of ARC, not ZEA.  Also, this particular header has 8 non-zero polynomial correction terms, which I initially figured meant 3rd order or so.  But the CGII paper says the corrections are only in the radial direction, so this is really a 7th order correction.

So I tried `FittedSIPWCS(..., wcs_type='ARC', order=7, center=south_pole)` (forcing a center was already an option :) ), but that didn't do any better :(.  2D order=7 isn't really the same anyway, since there are 14 extra degrees of freedom, and the 2D polynomial can't do odd powers of r.